examples/inputs/legacy_example_input.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
I'm wondering if there's a way to demo this without including a legacy input file in the example inputs. In v2 we had a lot of [input files](https://github.com/NREL/floris/tree/v2/examples/other_jsons) and the differences between them were lost due to noise, so I'd like to only add another one here when we definitely need it. For this use case, would it be enough to include a script in the docstring for the new class?

Reply:
Yeah sounds good to me! I'll work on getting that in.